console-timber-tree/src/main/java/com/jraska/console/timber/ConsoleTree.java

```diff
@@ -14,182 +20,203 @@
 
 public final class ConsoleTree extends Timber.Tree {
 
-  //region Constants
-
-  private static final int PLACEHOLDER = 0;
-  private static final int COLOR_VERBOSE = 0xff909090;
-  private static final int COLOR_DEBUG = 0xffc88b48;
-  private static final int COLOR_INFO = 0xffc9c9c9;
-  private static final int COLOR_WARN = 0xffa97db6;
-  private static final int COLOR_ERROR = 0xffff534e;
-  private static final int COLOR_WTF = 0xffff5540;
-
-  private static final int[] DEFAULT_COLORS = {PLACEHOLDER, PLACEHOLDER, COLOR_VERBOSE, COLOR_DEBUG,
-      COLOR_INFO, COLOR_WARN, COLOR_ERROR, COLOR_WTF};
-  private static final int REQUIRED_COLORS_LENGTH = DEFAULT_COLORS.length;
+    public static final DateFormat DEFAULT_TIME_FORMAT = new SimpleDateFormat("HH:mm:ss", Locale.US);
+    //region Constants
 
-  private static final int CALL_STACK_INDEX = 6;
-  private static final Pattern ANONYMOUS_CLASS = Pattern.compile("(\\$\\d+)+$");
+    private static final int PLACEHOLDER = 0;
```

Comment:
Please check your indentation. The whole file has changes, because you use 4 spaces instead of 2 in this project.